dotCMS/src/main/java/com/dotcms/rest/api/v1/categories/CategoryHelper.java

```diff
@@ -1,12 +1,17 @@
 package com.dotcms.rest.api.v1.categories;
 
+import com.dotcms.rest.api.v1.DotObjectMapperProvider;
```

Comment:
Are these imports necessary? If not, let's roll these changes back.

Reply:
Let me remove unnecessary imports

Reply:
code cleanup done @jcastro-dotcms